### DISCUSSION TITLE
[Bug]: Unbearable city ambient sounds

### DISCUSSION THREAD

**Discussion by Shituation** [OpenTTD/OpenTTD#11731] [Features]
### Version of OpenTTD

12.1 / Windows 10

### Expected result

Can you change the ambient sounds behavior? make them trigger less often? 
Late game needs to be muted because sawmills and docks drives anyone crazy in minutes.

### Actual result

Having to mute ambient sounds.

### Steps to reproduce

Play with ambient sounds on.

**Comment by 2TallTyler** [comment]
Related (possible duplicate of): #8957

**Comment by TrueBrain** [comment]
Not really a bug, but a feature request. Being able to mute certain sounds feels like an excellent idea to implement :D Or rate-limit the same sound from happening .. something creative is needed here :D

**Comment by nielsmh** [comment]
Perhaps make the `SndPlayFx` family of functions also take a "sound category" flag, one of which could be "industry ambient sound". That would let the sound player rate limit ambient sounds and the like.

**Comment by Shituation** [comment]
Just wanted to add the vehicle sounds to the mix as well... ships can make anyone crazy in seconds. 
Maybe adding the sound effects as a dedicated submenu with individual settings available to switch on-off... just throwing ideas.

**Comment by andythenorth** [comment]
I just do this... 😄 
<img width="518" alt="image" src="https://user-images.githubusercontent.com/1780327/162568522-c56121a1-e56e-4bbf-8bd0-9f82c4715322.png">
